### DISCUSSION TITLE
How to get autocomplete postcss config by types?

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [postcss/postcss#1927] [General]
Hi. I trying use typing Config with jsdoc, but postcss-load-config not export type and how to typo config?

```js
/** @type {import('postcss-load-config').Config} */
module.exports = {
  ident: 'postcss-scss',
  syntax: 'postcss-scss',
  plugins: {
    autoprefixer: { grid: 'autoplace' },
  }
}
```

**Comment by ai** [comment]
It should work. What problem do you have?

**Reply by popuguytheparrot** [reply]
config have is any type 
<img width="438" alt="Снимок экрана 2024-02-13 в 16 33 07" src="https://github.com/postcss/postcss/assets/19804652/2845e3be-f3c1-417c-8aee-53b2f622b6ab">

**Reply by ai** [reply]
Maybe the syntax is wrong (try other @-commands).

Sorry, I don't use JSDocs.

**Reply by popuguytheparrot** [reply]
for tailwind config this syntax is correct

**Reply by icyJoseph** [reply]
@popuguytheparrot had you installed `postcss-load-config`?
